ee/modules/110-istio/templates/control-plane/iop.yaml

```diff
@@ -140,7 +140,8 @@ spec:
       proxy:
         image: {{ include "helm_lib_module_image" (list $ (printf "proxyv2%s" $imageSuffix )) }}
         clusterDomain: {{ $.Values.global.discovery.clusterDomain | quote }}
-        resources: {}
```

Comment:
What is the rendered resources section if the resourcesManagement isn't set?

Reply:
The default value ee/modules/110-istio/openapi/config-values.yaml in resourcesManagement

Reply:
It is obvious. But rendered `resources` section not.

Reply:
When I checked IstioOperator fields were filled with default values if I didn't specify them.

I think we should add these cases to the tests.
https://github.com/deckhouse/deckhouse/blob/40d1e17358e64f7f8b853b2a75b177d0064c5d3e/ee/modules/110-istio/template_tests/module_test.go